src/app/(Page)/page.tsx

```
@@ -0,0 +1,108 @@
+          <h2 className="mb-12 text-3xl font-bold text-center md:text-4xl">
+            注目のポケモン
+          </h2>
+          <div className="grid grid-cols-2 gap-8 md:grid-cols-4">
+            {pokemonList.map((pokemon) => (
+              <Link href={`pokemon/${pokemon.id}`}>
+                <div
+                  key={pokemon.id}
+                  className="overflow-hidden transition-shadow duration-300 bg-white rounded-lg shadow-md hover:shadow-xl"
+                >
+                  <Image
+                    src={pokemon.image || ""}
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)